<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>803</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelYourApiKey">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>660</y>
      <width>51</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>API Key:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelApiKey">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>660</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelTextToSend">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>660</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Text to send:</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>660</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0,0" columnstretch="0,0">
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxStyle"/>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxModel"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelSize">
       <property name="text">
        <string>Size:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelModel">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxSize"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="comboBoxQuality"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelQuality">
       <property name="text">
        <string>Quality:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelStyle">
       <property name="text">
        <string>Style:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>690</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>690</y>
      <width>111</width>
      <height>98</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonSaveAs">
       <property name="text">
        <string>Save &amp;as..</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSendText">
       <property name="text">
        <string>&amp;Send</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApiKey">
       <property name="text">
        <string>Change API &amp;Key</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>642</width>
      <height>642</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutImage">
     <item>
      <widget class="QLabel" name="labelImage">
       <property name="maximumSize">
        <size>
         <width>640</width>
         <height>640</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resources.qrc">:/assets/default-screen.png</pixmap>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>663</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
